pom.xml

```diff
@@ -1,119 +1,130 @@
+      <timezone>Australia/Sydney</timezone>
     </developer>
     <developer>
       <id>janb</id>
       <name>Jan Bartel</name>
-      <email>janb@webtide.com</email>
-      <organization>Webtide, LLC</organization>
-      <organizationUrl>https://www.webtide.com</organizationUrl>
-      <timezone>10</timezone>
+      <email>janb@apache.org</email>
+      <url>http://www.mortbay.com/people/janb</url>
+      <organization>Mort Bay Consulting</organization>
+      <organizationUrl>http://www.mortbay.com</organizationUrl>
+      <timezone>Australia/Sydney</timezone>
     </developer>
     <developer>
       <id>sbordet</id>
       <name>Simone Bordet</name>
       <email>simone.bordet@gmail.com</email>
-      <timezone>1</timezone>
+      <organization>Webtide, LLC</organization>
+      <organizationUrl>https://webtide.com</organizationUrl>
+      <timezone>Europe/Rome</timezone>
     </developer>
     <developer>
-      <id>jesse</id>
-      <name>Jesse McConnell</name>
-      <email>jesse.mcconnell@gmail.com</email>
+      <id>lorban</id>
+      <name>Ludovic Orban</name>
+      <email>lorban@bitronix.be</email>
       <organization>Webtide, LLC</organization>
-      <organizationUrl>https://www.webtide.com</organizationUrl>
-      <timezone>-6</timezone>
+      <organizationUrl>https://webtide.com</organizationUrl>
+      <timezone>Europe/Brussels</timezone>
     </developer>
     <developer>
-      <id>djencks</id>
-      <name>David Jencks</name>
-      <email>david_jencks@yahoo.com</email>
-      <organization>IBM</organization>
-      <timezone>-8</timezone>
+      <id>olamy</id>
+      <name>Olivier Lamy</name>
+      <email>oliver.lamy@gmail.com</email>
+      <organization>Webtide, LLC</organization>
+      <organizationUrl>https://webtide.com</organizationUrl>
+      <timezone>Australia/Brisbane</timezone>
     </developer>
     <developer>
-      <id>joakime</id>
-      <name>Joakim Erdfelt</name>
-      <email>joakim.erdfelt@gmail.com</email>
-      <!-- other email addresses
-           joakim@erdfelt.com
-           joakim@intalio.com
-           joakim@webtide.com
-        -->
+      <id>jesse</id>
+      <name>Jesse McConnell</name>
+      <email>jesse@webtide.org</email>
       <organization>Webtide, LLC</organization>
-      <organizationUrl>https://www.webtide.com</organizationUrl>
-      <timezone>-7</timezone>
+      <organizationUrl>https://webtide.com</organizationUrl>
+      <timezone>America/Chicago</timezone>
     </developer>
     <developer>
-      <id>tbecker</id>
-      <name>Thomas Becker</name>
-      <email>thomas.becker00@googlemail.com</email>
-      <timezone>1</timezone>
+      <id>joakime</id>
+      <name>Joakim Erdfelt</name>
+      <email>joakim@webtide.com</email>
+      <organization>Webtide, LLC</organization>
+      <organizationUrl>https://webtide.com</organizationUrl>
+      <timezone>America/Chicago</timezone>
     </developer>
   </developers>
```

Comment:
I was hoping to list all the active developers, I think we are missing lachlan.  any others?

Reply:
Lachlan added